File: day17/src/com/am/Ex08.java
package com.am;

import java.io.File;

public class Ex08 {
	public static void main(String[] args) {
		File file = new File("target.txt"); //객체 잡고
		File result = new File("target2.txt"); //없는 객체 잡아서
		boolean boo = file.renameTo(result); //이름 바꾸기
		System.out.println(boo);
	}
}

File: stsWorkSpace/node01/ex08.js
function f1(a,b){console.log('test',a,b);return 1234;}
let f2=(a,b)=>{console.log('test',a,b);return 1234;};
let f3=(a,b)=>{return 1234;};
let f4=(a,b)=>4321;
let f5=(a)=>4321;
let f6=a=>4321;
let f7=()=>4321;
let result=f6();
console.log(result);